index.js

```diff
@@ -35,7 +35,7 @@ var printIssue = function(fileName, lineNum, matchedString){
 
 var findFixmes = function(file){
   // Prepare the grep string for execution (uses BusyBox grep)
-  var grepString = "grep -inHwoE " + fixmeStrings + " " + file;
+  var grepString = "grep -inHwoE " + fixmeStrings + " \"" + file + "\"";
```

Comment:
@ABaldwinHunter You're right, we're joining enough bits now that it's not very readable. Will change in a sec.